std/utf.d

```diff
@@ -1065,6 +1065,16 @@ if (isSomeChar!C)
 /// Whether or not to replace invalid UTF with $(LREF replacementDchar)
 alias UseReplacementDchar = Flag!"useReplacementDchar";
 
+// Reduce distinct instantiations of decodeImpl.
+private template CastForDecode(T)
+{
+    static if (isDynamicArray!T && is(T : E[], E) && __traits(isArithmetic, E)
+            && !is(E == shared))
+        alias CastForDecode = const(Unqual!E)[];
```

Comment:
Honestly, this seems pretty convoluted, though all of the other ways I can think of doing it would make it considerably longer. However, at least please make the `static if ` a single line so that it's not broken up without braces.

Reply:
Done.